includes/wikia/api/ArticlesApiController.class.php

```diff
@@ -983,7 +983,12 @@ public function getAsJson() {
 			}
 
 			if ( $redirect !== 'no' && $article->getPage()->isRedirect() ) {
-				$article = Article::newFromTitle( $article->getPage()->followRedirect(), RequestContext::getMain() );
+				// this will be equal to false if redirect is not local
```

Comment:
If it's local - it returns string with title.
If it's not local - it returns `false`.

I want this condition to be satisfied when it's string. Is it better if I use `is_string( $followRedirect )` or leave it as it is?